tony-core/src/main/java/com/linkedin/tony/Utils.java

```diff
@@ -404,4 +403,8 @@ public static void addResource(String path,
       LOG.error("Failed to add " + path + " to local resources.", exception);
     }
   }
+
+  public static String generateFileName(TonyApplicationMaster.Metadata obj) {
```

Comment:
I suggest a more descriptive name for the parameter such as `metadata` rather than `obj`.

Reply:
I was thinking about `metadataObj`, but I feel like any other name would add verbosity without additional value in terms of readability because the object type already conveys the attribute of this object. Do you have any other names in mind?

Reply:
Yeah, `metadataObj` is fine. I think it's still helpful to use a descriptive name despite the type since in the method body, you only see the variable name and not the type.

Reply:
Ok sure. Fixed: https://github.com/linkedin/TonY/pull/79/commits/25e79cf0094f5ed38caa4ef76db2addc04b540d0